pkg/exchange/ftx/exchange.go

```diff
@@ -179,7 +179,72 @@ func (e *Exchange) QueryAccountBalances(ctx context.Context) (types.BalanceMap,
 	return balances, nil
 }
 
+var supportedIntervals = map[types.Interval]int{
+	types.Interval1m:  1,
+	types.Interval5m:  5,
+	types.Interval15m: 15,
+	types.Interval1h:  60,
+	types.Interval1d:  60 * 24,
+	types.Interval3d:  60 * 24 * 3,
```

Comment:
30m and 6h, 12h are not supported?

Reply:
yes ,

according to the document

https://docs.ftx.com/#get-historical-index
resolution field
//window length in seconds. options: 15, 60, 300, 900, 3600, 14400, 86400, or any multiple of 86400 up to 30*86400

which means it only support the following interval
0.25m,1m,5m,15m,1h,4h,1d,2d...30d